swap_meet/item.py

```diff
@@ -1,2 +1,28 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category = "", condition = 0, age = 0):
+        
+        self.category = category
+        self.condition = condition
+        self.age = age
+        
+
+    def __str__(self):
+        return "Hello World!"
+
+    def condition_description(self):
```

Comment:
Good job making an if/elif statement here. I would look into how you could possibly improve this block of code by using other data structures.